documentation/airport-guides/cyhz.md

```diff
@@ -0,0 +1,150 @@
+|"HALIFAX ATIS"       | 118.025MHz |                                               |
+
+### About the Halifax Terminal Control Areas
+NAVCanada outlines the airspace boundary around Halifax in their Designated Airspace Handbook. All radar positions (departure, arrival) surrounding Halifax have the callsign "Halifax Terminal". Aircraft leaving the Halifax TCA should be handed over to Moncton Centre.
+
+[This diagram](http://prntscr.com/hv4bf4) may assist in visualising the airspace around Halifax.
+
+### Instrument Procedures
+Halifax has one Standard Instrument Departure and four Standard Terminal Arrival Routes.
+
+##### Departures
+All departing aircraft will be assigned the Halifax Four departure. The Halifax Four advises pilots to climb to 5000ft on runway heading. Upon receipt of an aircraft on departure, Halifax Terminal should: 
+* give a direct to a point along the filed route;
+* give vectors to a point along the filed route;
+* issue climb instructions.
+
+##### Arrivals
+Moncton Centre will assign STARs to aircraft and will give initial descent instructions.
+
+Halifax Terminal is responsible for the following:
+* issuing further descent clearances;
+* clearing aircraft onto a final approach, ILS, RNAV, Visual or LOC;
+* ensuring aircraft have sufficient spacing between them.
+
+Where possible, Terminal should issue descent clearances to meet the following restrictions:
+- Runway 05
+  * 5000' at PEPTA
+- Runway 14
+  * 5000' at VIMLA
+  * 5000' at ILOTA
+- Runway 23
+  * 5000' at RUDBU
+  * 5000' at PEPTA
+  * 5000' at LOGPO
+- Runway 32
+  * 5000' at MENVO
+  * 5000' at EBLUV
+  * 5000' at NOTUB
+
+The following fixes are Final Approach Course Fixes, meaning that aircraft can be cleared for an approach after these fixes:
+* ODKAS -> LOC, NDB or RNAV 05
+* TETAR -> ILS or RNAV 14
+* LEROS -> ILS, RNAV or NDB 23
+* VOKIL -> RNAV 32
+
+There are various transitions which can bring aircraft from an Initial Approach Fix to a FACF or onto approach:
+| Runway | Type of Approach |       Transition names     |
+|--------|:----------------:|:--------------------------:|
+|   05   |   LOC, RNAV(Z)   |        OBNOM, HOROW        |
+|        |     RNAV (Y)     |        PEPTA, GOSUG        |
+|   14   |   ILS, RNAV(Z)   |        MODIR, IGVOP        |
+|        |     RNAV (Y)     |        ILOTA, VIMLA        |
+|   23   |   ILS, RNAV(Z)   |        DUTIR, OTVUV        |
+|        |     RNAV (Y)     |        LOGPO, RUDBU        |
+|   32   |     RNAV (Z)     |        EMBAX, GABAP        |
+|        |     RNAV (Y)     | NOTUB, EBLUV, MENVO, AVRIX |
+
+The correct phraseology for clearing an aircraft onto approach via a transition is:
+"XXXXX, cleared APPROACH runway NUMBER, via TRANISTION"
+
+Substitute:
+- APPROACH for ILS, LOC, RNAV, visual
+- NUMBER for runway number
+- Transition for transition name
+
+
+eg. *Encore 3423, cleared ILS approach runway 23 via DUTIR*
+
+
+### Sectors Available in OpenScope

```

Comment:
The sectors here aren't referring to the the different areas of the airspace, but rather the controller positions found within it, so this should be the same as the positions section you have above.